frontend/src/routes/manage/Realm/Content/Edit/RemoveButton.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useRef } from "react";
+import { useTranslation, Trans } from "react-i18next";
+import { useFragment, graphql, useMutation } from "react-relay";
+import { FiTrash } from "react-icons/fi";
+
+import type { RemoveButtonData$key } from "../../../../../query-types/RemoveButtonData.graphql";
+import type { RemoveButtonMutation } from "../../../../../query-types/RemoveButtonMutation.graphql";
+import { ConfirmationModal, ConfirmationModalHandle } from "../../../../../ui/Modal";
+import { displayCommitError } from "../../util";
+import { Button } from "../util";
+
+
+type Props = {
+    block: RemoveButtonData$key;
+    onConfirm?: () => void;
+};
+
+export const RemoveButton: React.FC<Props> = ({ block: blockRef, onConfirm }) => {
+    const { t } = useTranslation();
+
+
+    const block = useFragment(graphql`
+        fragment RemoveButtonData on Block {
+            id
+        }
+    `, blockRef);
+
+
```

Comment:
Well, both the stuff above and below it have a different "topic." It's either both, or just one line on both sides (and everywhere else in that function). Like, double-newline is used to group things that potentially have "subgroups" separated by single newlines. If there was just a single newline on either end, it would mark this snippet as belonging to the corresponding group, which is not what I intended to express.

Reply:
:shrug: